lib/vc.js

```diff
@@ -65,6 +65,22 @@ module.exports = {
   dateRegex
 };
 
+/**
+ * @typedef {Object} VerifyPresentationResult
```

Comment:
Run lint and fix doc issues.

Reply:
I fixed the issues related to this PR. There are other linting issues that existed that are unrelated.